### PR TITLE
SRIOV: add new case for netkvm protocal binding on windows

### PR DESCRIPTION
**Pull request by kimi1978wy**
Binding of netkvm protocol to netkvm adapter and enabled manually without VF.

id: 1927119

Signed-off-by: Yu Wang <wyu@redhat.com>

### CODE REVIEW

**Comment by kimi1978wy** [comment]
@yanglei-rh @PaulYuuu Could you help to check this patch?

 (1/3) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.i386.io-github-autotest-qemu.netkvm_protocol_binding: PASS (182.33 s)
 (2/3) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.netkvm_protocol_binding: PASS (179.55 s)
 (3/3) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.netkvm_protocol_binding: PASS (236.00 s)
 (1/4) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.netkvm_protocol_binding: PASS (93.54 s)
 (2/4) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.netkvm_protocol_binding: PASS (236.51 s)
 (3/4) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.i386.1.io-github-autotest-qemu.netkvm_protocol_binding: PASS (109.55 s)
 (4/4) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.x86_64.1.io-github-autotest-qemu.netkvm_protocol_binding: PASS (132.64 s)

**Comment by yanglei-rh** [comment]
Hi, @kimi1978wy I suggest keeping the title of the patch and the title of the bug the same, and from the content point of view, this seems to have little to do with SRIOV. Also please add "only virtio_net" in the text. Other content is okay for me.

**Comment by kimi1978wy** [comment]
> Hi, @kimi1978wy I suggest keeping the title of the patch and the title of the bug the same, and from the content point of view, this seems to have little to do with SRIOV. Also please add "only virtio_net" in the text. Other content is okay for me.

Thanks, will update it

**Comment by yanglei-rh** [comment]
LGTM

Test Pass:
(1/1) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.netkvm_protocol_binding.q35: PASS (213.83 s)
(1/1) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.i386.io-github-autotest-qemu.netkvm_protocol_binding.q35: PASS (356.91 s)
(1/1) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.netkvm_protocol_binding.q35: PASS (277.58 s)
(1/1) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.netkvm_protocol_binding.q35: PASS (123.52 s)
(1/1) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.netkvm_protocol_binding.q35: PASS (107.88 s)
(2/2) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.Win10.i386.io-github-autotest-qemu.netkvm_protocol_binding.q35: PASS (129.29 s)
(1/1) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.netkvm_protocol_binding.q35: PASS (178.96 s)
(1/1) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.netkvm_protocol_binding.q35: PASS (201.82 s)
(2/2) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.i386.io-github-autotest-qemu.netkvm_protocol_binding.q35: PASS (214.99 s)

File: qemu/tests/netkvm_protocol_binding.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    3) Install VIOPROT protocol
+    4) Bind netkvm protocol to netkvm adapter
+    5) Ping out
+
+    :param test: QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+
+    timeout = int(params.get_numeric("timeout", 360))
```
Comment (PaulYuuu) [inline qemu/tests/netkvm_protocol_binding.py:25]:
```suggestion
    timeout = params.get_numeric("timeout", 360)
```

**Review by PaulYuuu — COMMENTED** [review]
Other LGTM.

**Review by PaulYuuu — APPROVED** [review]

**Pull request merged**